layers/+lang/common-lisp/packages.el

```diff
@@ -33,6 +34,10 @@
 (defun common-lisp/post-init-helm-gtags ()
   (spacemacs/helm-gtags-define-keys-for-mode 'common-lisp-mode))
 
+(defun common-lisp/init-slime-company ()
+  :config
```

Comment:
Besides there's no need to delay setting variables until the defvar has happened, for that matter.